Tests/ML/visualizers/test_plot_cross_validation.py

```diff
@@ -310,14 +312,22 @@ def test_result_aggregation_for_classification_all_epochs(test_output_dirs: Outp
                                                 expected_aggregate_metrics=expected_aggregates,
                                                 expected_epochs={1, 2, 3})
 
+
 @pytest.mark.after_training_ensemble_run
-def test_add_comparison_data(test_config_comparison: PlotCrossValidationConfig) -> None:
-    test_config_comparison.epoch = 2
-    metrics_df, root_folder = download_metrics(test_config_comparison)
```

Comment:
Isn't it better to modify the test_config_comparison test fixture? It's only used here. Or at least delete it

Reply:
good catch! deleted.